doc/whats-new.rst

```diff
@@ -25,6 +25,8 @@ Breaking changes
 
 New Features
 ~~~~~~~~~~~~
+- Added `~xarray.DataArray.cov` and `~xarray.DataArray.corr` (:pull:`3550`).
```

Comment:
Good, though these should have `:py:meth:` before them (some other examples below)

Reply:
Thanks! That's already enough?

Reply:
What do you mean by already enough?

Reply:
If changes have been made only for `xarray.dataarray.cov` and `.corr`, the above-mentioned changes in `whats-new.rst` are sufficient?

Reply:
Yes! You could maybe say what the functions do, but they're already fairly self-explanatory (and we'll need to change this a bit given the updated location)

Reply:
Alright 🙂 

Yes, I wanted to already understand it completely. Will update the merge request later today. With the moved functions.